### ISSUE TITLE
Increase contrast ratio of `yellow`

### ISSUE THREAD

**Issue by Bikossor** [mondersky/tabscolor-vscode#1]
First of all, thank you for this extension!

I'd like to make a suggestion to increase the contrast ratio of the `yellow`  color. The foreground has currently the css color `brown` which has a contrast ratio of `4.26` to the background. If the foreground color would be `black` the contrast ratio would be `12.64` and much more readable 😄

**Comment by mondersky** [comment]
You're welcome, 
in fact the contrast in the yellow color isn't optimal, I have changed the font color to black. Thank you for your suggestion :)

**Comment by Bikossor** [comment]
If you need some help in the future with this extension then let me know 😄

**Comment by mondersky** [comment]
Thank you for your interest in the extension, your help is welcome. You can either pick one of the TODO tasks ( find the full list in the extension page) or help debug one of the open issues.

**Comment by Bikossor** [comment]
Will do!
